README.md

```diff
@@ -1,12 +1,16 @@
 # Kafka Connect BigQuery Connector
 
 [![Source](https://img.shields.io/badge/source-wepay/kafka–connect–bigquery-blue.svg?style=flat-square)](https://github.com/wepay/kafka-connect-bigquery)
-[![Release](https://img.shields.io/github/release/wepay/kafka-connect-bigquery.svg?style=flat-square)](https://travis-ci.org/wepay/kafka-connect-bigquery)
+[![Release](https://img.shields.io/github/release/wepay/kafka-connect-bigquery.svg?style=flat-square)](https://github.com/wepay/kafka-connect-bigquery/releases)
+[![Maven Central](https://img.shields.io/maven-central/v/com.wepay/kafka-connect-bigquery.svg?style=flat-square)](https://repo1.maven.org/maven2/com/wepay/kafka-connect-bigquery/)
```

Comment:
Sorry to change this on you, but would you mind switching the link to http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.wepay%22%20AND%20a%3A%22kafka-connect-bigquery%22 ? I noticed Debezium was doing something similar and think it probably makes more sense to lead to the Maven Central version page for the connector than the actual Maven Central repo.